snomed/com.b2international.snowowl.snomed.datastore/src/com/b2international/snowowl/snomed/datastore/converter/ModuleExpander.java

```diff
@@ -46,21 +44,15 @@ void expand(List<? extends SnomedComponent> results) {
 		if (expand.containsKey(SnomedComponent.Expand.MODULE)) {
 			final Options moduleOptions = expand.get(SnomedComponent.Expand.MODULE, Options.class);
 
-			final Set<String> moduleIds = results.stream().map(SnomedComponent::getModuleId).collect(Collectors.toSet());
+			final SortedSet<String> moduleIds = results.stream().map(SnomedComponent::getModuleId).collect(ImmutableSortedSet.toImmutableSortedSet(String::compareTo));
```

Comment:
This one could also be an `Iterable` for consistency, since `SnomedConceptRequestCache` takes care of making the input unique.

In the converter classes below, `Iterable`s and `Stream`s are about equal in numbers as far as I can tell; would you like to harmonize those into a One True Way of collecting concept IDs off components? 😄 Then whichever type is chosen, that could be used as the first parameter in `SnomedConceptRequestCache#request`.

Reply:
Done